src/ComfortPay/Message/ListOfExpirePerIdRequest.php

```diff
@@ -37,17 +37,21 @@ public function sendData($data)
             }, $data['cardIds']));
         }
 
+        $request = new \stdClass();
+        $request->listOfIdCards = implode(',', $data['cardIds']);
```

Comment:
Are you sure about imploding this? There's nothing about imploding `cardIds` in the docs, and WSDL states, that the number of elements is unbounded:

```xml
<element name="getListOfExpPerId">
   <complexType>
    <sequence>
     <element name="listOfIdCards" type="xsd:string" minOccurs="1" maxOccurs="unbounded"/><!-- teleplatba__getListOfExpPerId::listOfIdCards -->
    </sequence>
   </complexType>
  </element>
```

Reply:
It works also with imploded cardIds but I changed to more elements of `listOfIdCards`